src/CoreGraphics/CGImage.cs

```diff
@@ -216,12 +216,17 @@ protected virtual void Dispose (bool disposing)
 		static extern IntPtr CGWindowListCreateImage(CGRect screenBounds, CGWindowListOption windowOption, uint windowID, CGWindowImageOption imageOption);
         
 		public static CGImage ScreenImage (int windownumber, CGRect bounds)
-		{                    
-			IntPtr imageRef = CGWindowListCreateImage(bounds, CGWindowListOption.IncludingWindow, (uint)windownumber,
+		{
+			return ScreenImage (windownumber, bounds, CGWindowListOption.IncludingWindow);
+		}
+
+		public static CGImage ScreenImage (int windownumber, CGRect bounds, CGWindowListOption windowOption)
```

Comment:
I wonder if the `imageOption` should not be exposed too ?

Reply:
Initially just added the window options because that's what I'm using :-) but I guess it would indeed be cleaner to expose both options in this overload. I'll update the pr if you want

Reply:
I think it make sense but you can wait for more comments and let the bot complete this first around of testing :)

Reply:
pushed an update including the imageOption